pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -232,15 +237,22 @@ func evictPods(inputPods []*v1.Pod,
 	totalCPU *float64,
 	totalMem *float64,
 	podsEvicted *int,
-	dryRun bool, maxPodsToEvict int) {
+	dryRun bool, maxPodsToEvict int, taintsOfLowNodes map[string][]v1.Taint) {
```

Comment:
Each parameter of `evictPods` lies on its own line. Please, do the same for `taintsOfLowNodes`.